doc/classes/MultiMesh.xml

```diff
@@ -89,6 +89,10 @@
 	</methods>
 	<members>
 		<member name="buffer" type="PackedFloat32Array" setter="set_buffer" getter="get_buffer" default="PackedFloat32Array()">
+			Contains data about each mesh instance. It's formatted in sets of 16 floats that look like this, followed by an empty set of 4:
```

Comment:
Format can be different depending on `use_colors`, `use_custom_data`, and used renderer (color and custom data cam be 4 floats or 4 half-floats packed to 2 floats).

See:

https://github.com/godotengine/godot/blob/b7feebefabc2d48b0d4794cd31fc141f1caecc5c/servers/rendering/renderer_rd/storage_rd/mesh_storage.cpp#L1425-L1432

https://github.com/godotengine/godot/blob/b7feebefabc2d48b0d4794cd31fc141f1caecc5c/drivers/gles3/storage/mesh_storage.cpp#L1472-L1481

Reply:
Okay, thank you. I'm not going to did too much deeper into it so I'll just close the request.